contact.html

```diff
@@ -26,6 +26,7 @@
         />
         </a>
         <ul>
+          <!-- <li></li>  -->
```

Comment:
What's this?

Reply:
For smaller screen it the modal to navigate to different pages. 
![a1](https://github.com/rbjks/rbjks.github.io/assets/115359380/a56183fc-bd7d-4168-abb7-c15fc5a75858)